linkerd/app/inbound/src/http/server.rs

```diff
@@ -127,9 +127,10 @@ impl<T: Param<tls::ConditionalServerTls>> ExtractParam<errors::respond::EmitHead
 
 impl errors::HttpRescue<Error> for ServerRescue {
     fn rescue(&self, error: Error) -> Result<errors::SyntheticHttpResponse> {
-        if let Some(cause) = errors::cause_ref::<crate::policy::DeniedUnauthorized>(&*error) {
+        if let Some(cause) = errors::cause_ref::<policy::HttpRouteUnauthorized>(&*error) {
```

Comment:
does this also want to handle the `ServerUnauthorized` error, or will that never make it here?

Reply:
we shouldn't ever hit that now, since HTTP authorization decisions are made per-request and not at the server level.